src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -127,7 +127,7 @@ class JesBackend extends Backend with LazyLogging {
       val evaluatedExpression = taskOutput.expression.evaluate(scopedLookupFunction, engineFunctions, interpolateStrings = true)
       evaluatedExpression match {
         case Success(v) =>
-          val jesOutput = JesOutput(taskOutput.name, s"$callGcsPath/${taskOutput.name}", Paths.get(v.valueString))
+          val jesOutput = JesOutput(taskOutput.name, s"$callGcsPath/${v.valueString}", Paths.get(v.valueString))
```

Comment:
Discussed in person but for posterity call outputs are written to the call's directory and the path is retained so two files with the same name won't conflict